### ISSUE TITLE
Update tutorial 4: topic name becomes a configuration parameter

### ISSUE THREAD

**Issue by JulienPeloton** [astrolabsoftware/fink#8]
As per https://github.com/astrolabsoftware/fink-broker/commit/61523a6bdcef85eebbe99b607d36a12b56ca35d5, the Kafka topic name for redistribution will become a configuration parameter. Instructions in `docs/tutorials/redistributing_alerts.md` must be updated once the PR is merged then.

**Comment by JulienPeloton** [comment]
Won't fix as the tutorial has been removed.
